py-scripts/test_l3.py

```diff
@@ -6171,6 +6172,8 @@ def change_port_to_ip(upstream_port, lfclient_host, lfclient_port):
     return upstream_port
 
 # Only used by argparser, so safe to exit in this function
```

Comment:
This is a dangling comment now.  Please make sure it stays attached to function, possibly can just move it below function signature in same commit.